react/components/ContentWrapper.js

```diff
@@ -30,24 +30,26 @@ class ContentWrapper extends Component {
       history,
       headerContent,
       namespace,
+      hideBackButton = false,
     } = this.props
     const { shouldShowError } = this.state
 
+    const backButtonConfigs = {
+      linkLabel: backButton
+        ? backButton.title
+          ? backButton.title
```

Comment:
Is that a typo? Looks like you're checking the same think twice.

Reply:
It's not! 
this translates to 
```
let linkLabel
if (backButton)
    linkLabel = backButton.title ? backButton.title : intl.formatMessage({ id: backButton.titleId })
else 
    linkLabel = intl.formatMessage({ id: 'commons.back' })
```